ppl/cmd/zar/find/command.go

```diff
@@ -109,21 +109,21 @@ func (c *Command) Run(args []string) error {
 		findOptions = append(findOptions, archive.SkipMissing())
 	}
 
+	ctx, cancel := context.WithCancel(context.Background())
```

Comment:
nit: make this context.TODO()

Reply:
Done in 921f5b6c7c5619c422e75072cf1d8b0f0d78220a.